platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/state/address/AddressBookUtils.java

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.swirlds.platform.state.address;
+
+import static com.swirlds.platform.util.BootstrapUtils.detectSoftwareUpgrade;
+
+import com.swirlds.common.context.PlatformContext;
+import com.swirlds.common.platform.NodeId;
+import com.swirlds.platform.state.MerkleRoot;
+import com.swirlds.platform.state.PlatformStateAccessor;
+import com.swirlds.platform.state.signed.ReservedSignedState;
+import com.swirlds.platform.system.SoftwareVersion;
+import com.swirlds.platform.system.address.AddressBook;
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/**
+ * Utility class for address book operations.
+ */
+public class AddressBookUtils {
```

Comment:
This creates a 3rd `AddressBookUtils.java` class.   Would it make sense to add this functionality to this one: https://github.com/hashgraph/hedera-services/blob/develop/platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/system/address/AddressBookUtils.java

Reply:
Yes didn't know those many existed :) . Moved to the place you suggested. Thanks @edward-swirldslabs